python/tests/test_account.py

```diff
@@ -24,6 +36,12 @@ def test_os_name(self, tmpdir):
         ac1 = Account(p.strpath, os_name="solarpunk")
         ac1.get_info()
 
+    def test_save_self_keypair(self, acfactory, datadir):
+        ac = acfactory.get_unconfigured_account()
+        ac._save_self_keypair("alice@example.com",
+                              datadir.join('key/alice-public.asc').read(),
+                              datadir.join('key/alice-secret.asc').read())
```

Comment:
not really a complete test -- you could run configure and verify that the pre-configured key is really used.

Reply:
I was originally going to write this test in test_lowlevel.  My reasoning for this test was that the rust function is already unit-tested and this test ensures the ffi function doesn't it's boring marshalling stuff by checking we can call this and not crash.  But I guess it could still be mangling the strings or so.

I want to follow up on this by integrating this new function into AccountMaker so that it uses the Alice and Bob keys when creating new accounts.  At this point it will be tested fully.  Do you think there's much value in building this test out more thoroughly in the mean time?